Dokan/Dokan/Localization/Generated/Strings+Generated.swift

```diff
@@ -0,0 +1,141 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
```

Comment:
You can write a comment that file is automatically generated like :
// this is automatic generated file please don't edit it 🔫

Reply:
okay ✔